script/erc5018-blob-test.js

```diff
@@ -0,0 +1,141 @@
+const { ethers } = require("hardhat");
```

Comment:
I think you may need to write the test under the test folder and use "chai" to check if the real result is the same as what you expect